conf/nginx.conf

```diff
@@ -70,4 +70,9 @@ server {
     rewrite ^/proof-of-work/proof-of-work/?$                             $scheme://$http_host/proof-of-work/overview permanent;
     rewrite ^/governance/introduction-to-decred-governance/?$            $scheme://$http_host/governance/overview permanent;
     rewrite ^/contributing/contributing-to-decred/?$                     $scheme://$http_host/contributing/overview permanent;
+    rewrite ^/advanced/address-details/?$                                $https://devdocs.jamieholdstock.com/developer-guides/addresses permanent;
```

Comment:
URLs need updating now that https://devdocs.decred.org is up

Reply:
Done